packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -100,7 +106,7 @@ export const Dropdown = forwardRef(
     ) => {
         const [isToggled, setIsToggledState] = useState(false);
         const [coords, setCoords] = useState<Coords>();
-        const [clickPos, setСlickPos] = useState<Coords>();
+        const [clickPos, setClickPos] = useState<Coords>();
```

Comment:
I have fixed this because auto-correct marked this as spelling mistake. 

The diff is this:
```
// Original: \xd0\xa1\x6c\x69\x63\x6b
// New:      \x43\x6c\x69\x63\x6b
```

Reply:
LOL, its this:

- `U+0421`	`С`	`\xd0\xa1`	`CYRILLIC CAPITAL LETTER ES`
- vs. `LATIN CAPITAL LETTER C' (U+0043)`

Reply:
No idea how it happend tbh 😓 Probably was rewriting. They are also on the same key.